### DISCUSSION TITLE
std::experimental::simd

### DISCUSSION THREAD

**Discussion by bernhardmgruber** [microsoft/STL#1714] [General] [answered]
GCC 11 will come with an experimental implementation of the `std::simd` data types, currently in the parallelism TS 2. I am super excited for this feature and wonder if there are any plans yet to also have an experimental implementation inside Microsoft's STL.

I know that you state in the Non-Goals section of your README that you do not plan to implement TSes, so it is a perfectly fine answer to say it's not on the roadmap ;)

**Comment by CaseyCarter** [comment, marked as answer]
This is not on our roadmap right now. I'd love to have resources to implement all of the library TSes but we don't currently have that kind of capacity. That said, I'm hopeful that as the open source community around the STL grows we may have the resources in our community to get things done that those of us on the VC Libs team can't convince our managers to fund.
